MdeModulePkg/Library/SmmLockBoxLib/SmmLockBoxStandaloneMmLib.inf

```diff
@@ -38,6 +38,7 @@
   MmServicesTableLib
   BaseLib
   DebugLib
+  StandaloneMmDriverEntryPoint

```

Comment:
Why a lib needs to depend on DriverEntryPoint?

Reply:
Hi Ray Thanks I've remove it in the lastest version.